docs/components/InputTime/Web.stories.tsx

```diff
@@ -26,13 +26,16 @@ const BasicTemplate: ComponentStory<typeof InputTime> = args => (
 );
 
 const EventTemplate: ComponentStory<typeof InputTime> = args => {
-  const [time, setTime] = useState(new CivilTime(3, 52));
+  const [time, setTime] = useState<CivilTime>();
+
   const resetTime = () => {
     setTime(new CivilTime(3, 52));
```

Comment:
I wonder since the default is no longer 3, 52 if we should change this to reflect that?

Reply:
Yep. Done. This will show up in storybook.